src/grz_cli/upload.py

```diff
@@ -128,11 +127,12 @@ def upload_file(self, local_file_path, s3_object_id):
         self.__log.info(f"Uploading {local_file_path} to {s3_object_id}...")
 
         filesize = getsize(local_file_path)
+        multipart_chunksize = self._config.s3_options.multipart_chunksize
 
         chunksize = (
             math.ceil(filesize / MULTIPART_MAX_CHUNKS)
-            if filesize / MULTIPART_CHUNKSIZE > MULTIPART_MAX_CHUNKS
-            else MULTIPART_CHUNKSIZE
+            if filesize / multipart_chunksize > MULTIPART_MAX_CHUNKS
```

Comment:
I prefer using parentheses for explicit operation ordering, but I won't die on that hill.

Reply:
Feel free to modify, but you will also have to convince ruff

Reply:
Eh, ruff can have it